cmd/agent/install_script.sh

```diff
@@ -133,6 +133,12 @@ if [ $OS = "RedHat" ]; then
     $sudo_cmd yum -y clean metadata
     $sudo_cmd yum -y --disablerepo='*' --enablerepo='datadog' install datadog-agent || $sudo_cmd yum -y install datadog-agent
 elif [ $OS = "Debian" ]; then
+
+    keyserver=hkp://keyserver.ubuntu.com:80
+    if [ -n "$USE_APT_BACKUP_KEYSERVER" ]; then
```

Comment:
we should avoid using env vars directly in the middle of the script, it makes it very hard to understand the vars that the script supports. Let's define a local variable like `use_apt_backup_keyserver` at the beginning of the script and then use it here.